src/status_im/contexts/wallet/create_account/select_keypair/view.cljs

```diff
@@ -63,16 +63,20 @@
       :details             {:full-name (:name item)
                             :address   (when (zero? index)
                                          (utils/get-shortened-compressed-key compressed-key))}
+      :on-press            #(set-selected-key-uid (:key-uid item))
       :accounts            accounts
-      :default-selected?   (zero? index)
+      :selected?           (= selected-key-uid (:key-uid item))
       :container-style     {:margin-horizontal 20
                             :margin-vertical   8}}]))
 (defn view
   []
   (let [{:keys [compressed-key customization-color]} (rf/sub [:profile/profile])
         profile-with-image                           (rf/sub [:profile/profile-with-image])
         keypairs                                     (rf/sub [:wallet/keypairs])
-        profile-picture                              (profile.utils/photo profile-with-image)]
+        selected-keypair                             (rf/sub [:wallet/selected-keypair-uid])
+        profile-picture                              (profile.utils/photo profile-with-image)
+        [selected-key-uid set-selected-key-uid]      (rn/use-state selected-keypair)]
+    (rn/use-mount #(rf/dispatch [:wallet/get-keypairs]))
```

Comment:
why do we do this on mount? should we not do on-navigate to? or even on wallet load?

Reply:
Yeah it is fetched in wallet/initialize, forgot to remove this line, will remove it